package.json

```diff
@@ -526,6 +566,24 @@
                     "default": true,
                     "description": "Remove spaces around = in function keywords.",
                     "scope": "window"
+                },
+                "julia.execution.inlineResults": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Show inline result bubbles.",
+                    "scope": "window"
+                },
+                "julia.execution.resultInREPL": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Show inline execution results in REPL.",
+                    "scope": "window"
+                },
```

Comment:
we may want to "unify" this, maybe ? I mean, if `inlineResutls` are `false`, we show results in REPL, otherwise show inline bubbles (as in Juno).
I happened to turn off both configurations and I a bit confused by that any result didn't pop up anywhere (well, silly me)

Reply:
Dunno, people might want both?

Reply:
okay, how about #1250 ?

Reply:
I like it.